src/main/java/com/sda/Warehouse/repositories/JpaProductRepository.java

```diff
@@ -16,3 +17,4 @@
     Page<Product> findByNameIgnoreCaseContainingOrDescriptionIgnoreCaseContaining(String name, String description, Pageable pageable);
```

Comment:
This record is unreadable (too long), replace it in SQL query by @Query annotation